sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.net6.0.cs

```diff
@@ -20,6 +20,19 @@ public partial class DownloadTransferValidationOptions
         public bool AutoValidateChecksum { get { throw null; } set { } }
         public Azure.Storage.StorageChecksumAlgorithm ChecksumAlgorithm { get { throw null; } set { } }
     }
+    public partial class ExpectContinueOptions
+    {
+        public ExpectContinueOptions() { }
+        public System.TimeSpan Backoff { get { throw null; } set { } }
```

Comment:
This name looks more like it's for a retry behavior than something that configures how long `Auto` stays on for.  Consider a name more like like `AutoEnabledInterval`?